### PR TITLE
20181128 ual

### PR DESCRIPTION
**Pull request by hecperleo**
Just added the necessary stuff to be able to run with grvc-ual.

### CODE REVIEW

File: architecture/launch/realUav_laptop_monitor.launch [hunk 1]
```diff
@@ -6,6 +6,6 @@
     
     <!-- #######################    Log   ############################# -->
     <node pkg="rosbag" type="record" name="rosbag_record_all"
-       args="record -O /home/mfaria/openTheHatch/bags/record_all.bag /tf /scan /mavros/global_position/local /mavros/global_position/global /occupied_cells_vis_array /free_cells_vis_array /ltstar_path /frontiers_reply /frontiers_request /ltstar_request /ltstar_reply /mavros/setpoint_position/local /position_log /mavros/local_position/pose"/>
+       args="record -O $(env HOME)/openTheHatch/bags/record_all.bag /tf /scan /mavros/global_position/local /mavros/global_position/global /occupied_cells_vis_array /free_cells_vis_array /ltstar_path /frontiers_reply /frontiers_request /ltstar_request /ltstar_reply /mavros/setpoint_position/local /position_log /mavros/local_position/pose"/>
```
Comment (margaridaCF) [inline architecture/launch/realUav_laptop_monitor.launch]:
```suggestion
       args="record -O $(env HOME)/openTheHatch/bags/record_all.bag /tf /scan /mavros/global_position/local /mavros/global_position/global /occupied_cells_vis_array /free_cells_vis_array /ltstar_path /frontiers_reply /frontiers_request /ltstar_request /ltstar_reply /mavros/setpoint_position/local /position_log /mavros/local_position/pose  /octomap_binary"/>
```

File: architecture/launch/sitl_laptop_collectData.launch [hunk 2]
```diff
@@ -24,60 +24,38 @@
...
-        <param name="sensor_model/hit"          value="0.74"/>
-        <param name="sensor_model/miss"         value="0.48464"/>
-        
-        <param name="resolution"                value="1" />
+        <param name="latch" value="false" />
+        <param name="publish_free_space" value="true"/>
+        <param name="sensor_model/max_range" value="50"/>
+        <param name="sensor_model/hit" value="0.74"/>
+        <param name="sensor_model/miss" value="0.48464"/>
+        <param name="resolution" value="1" />
```
Comment (margaridaCF) [inline architecture/launch/sitl_laptop_collectData.launch]:
```suggestion
        <param name="resolution" value="0.5" />
```

File: architecture/launch/sitl_laptop_collectData.launch [hunk 3]
```diff
@@ -24,60 +24,38 @@
...
-    <node name="state_manager"      type="state_manager_node"   pkg="architecture"  output="screen" />
-    <!-- <node name="frontier_cells"     type="frontiers_async_node" pkg="frontiers"     output="screen" /> --> 
-    <node name="middle_man_current_position" type="current_position_provider_node" pkg="architecture"  output="screen" />
- 
+    <!-- <node name="lazy_theta_star" type="ltStar_async_node" pkg="lazy_theta_star" output="screen" /> -->
+    <!-- <node name="state_manager" type="state_manager_node" pkg="architecture" output="screen" /> -->
+    <!-- <node name="frontier_cells" type="frontiers_async_node" pkg="frontiers" output="screen" /> -->
+    <node name="middle_man_current_position" type="current_position_provider_node" pkg="architecture" output="screen" />
+    <!-- #######################  Save Poses Node  ############################# -->
+    <node name="save_poses" type="save_poses_node" pkg="architecture" output="screen" required="true"/>
```
Comment (margaridaCF) [inline architecture/launch/sitl_laptop_collectData.launch]: What is this node save_poses_nodes?
I'm not seeing any new python or cpp files.
Not sure if there is a file missing or the line shouldn't be there.
Was this to record the information in the pdf?

Comment (hecperleo) [reply architecture/launch/sitl_laptop_collectData.launch]:
```suggestion
    <node name="save_poses" type="save_poses_node" pkg="architecture" output="screen" required="true"/>
```

This line should not be there.

**Comment by hecperleo** [comment]
Ok, I will follow the technique.

**Pull request closed without merging**